### PR TITLE
Allow console trace id be set via env vars

### PR DESCRIPTION
**Pull request by frankdekker**
Allow TraceId to be picked up based on an ENV_VAR.

### CODE REVIEW

File: src/Service/TraceContext/TraceContextService.php [hunk 1]
```diff
@@ -43,10 +41,22 @@ public function createNewTrace(): TraceContext
         return $trace;
     }
 
+    public function createTraceFrom(string $traceId): TraceContext
```
Comment (bram123) [inline src/Service/TraceContext/TraceContextService.php]: Naming; what you're passing is not Just the traceId. But the full version-traceid-transactionid-flag string

File: tests/Unit/Service/TraceContext/TraceContextServiceTest.php [hunk 2]
```diff
@@ -54,6 +54,29 @@ public function testCreateNewTrace(): void
...
+        $trace = $this->service->createTraceFrom('invalid');
+        static::assertSame('00', $trace->getVersion());
+        static::assertSame('0af7651916cd43dd8448eb211c80319c', $trace->getTraceId());
+        static::assertSame('b7ad6b7169203331', $trace->getTransactionId());
+        static::assertSame('00', $trace->getFlags());
+    }
+
+    public function testCreateTraceFrom(): void
+    {
+        $this->generator->expects(static::once())->method('generateTransactionId')->willReturn('b7ad6b7169203331');
```
Comment (bram123) [inline tests/Unit/Service/TraceContext/TraceContextServiceTest.php]: Test with a different transactionId vs input transactionId in createTraceFrom

File: tests/Unit/Service/TraceContext/TraceContextServiceTest.php [hunk 3]
```diff
@@ -54,6 +54,29 @@ public function testCreateNewTrace(): void
...
+
+        $trace = $this->service->createTraceFrom('00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-00');
```
Comment (bram123) [inline tests/Unit/Service/TraceContext/TraceContextServiceTest.php:73]: Assert parentTransactionId

**Review by bram123 — CHANGES_REQUESTED** [review]

File: src/Service/TraceContext/TraceContextService.php (hunk 1 above)
Comment (frankdekker) [reply src/Service/TraceContext/TraceContextService.php]: Renamed it to `$traceParent` similar to the variable in called methods.

File: src/Service/TraceContext/TraceContextRenderer.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+namespace DR\SymfonyTraceBundle\Service\TraceContext;
+
+use DR\SymfonyTraceBundle\TraceContext;
+
+class TraceContextRenderer
+{
+    public static function renderTraceParent(TraceContext $trace): string
```
Comment (frankdekker) [inline src/Service/TraceContext/TraceContextRenderer.php:10]: Created separated renderer to allow rendering the TraceContext to string outside the library.

**Review by bram123 — APPROVED** [review]

**Pull request merged**